themes/theme-gmd/components/AppBar/presets/DefaultBar/connector.js

```diff
@@ -17,8 +17,8 @@ function makeMapStateToProps() {
  * @returns {Object}
  */
 const mapDispatchToProps = dispatch => ({
-  updateStatusBar: ({ background }) => {
-    dispatch(updateStatusBarBackground(background));
+  updateStatusBar: ({ background }, setDefault) => {
```

Comment:
I'd suggest calling this `isDefault`, because otherwise this looks like a function being passed as param.